LiteFarmOrg/LiteFarm
# LF-3055: Replace Unit component for % with Input component in WaterUsageCalculatorModal

**Description**

- Replace `<Unit />` used for `% with Input component` with `<Input />` in WaterUsageCalculatorModal. (The unit component is not designed to be used for percentage)
- pass `errors` to `WaterUsageCalculatorModal` from its parent component.
- delete `percentage_location` object which should not be used.

Note: You will see a warning in the console when you open the water use calculator modal, and it is going to be addressed in this [ticket](https://lite-farm.atlassian.net/browse/LF-3080). (it is not related to the input component)

Jira link: https://lite-farm.atlassian.net/browse/LF-3055

**Type of change**


**How Has This Been Tested?**


1. go to /tasks
2. "+ Create a task"
3. Select “Irrigate”
4. "Continue" until you see "Tell us about the Irrigation work task"
5. Select "Depth" for “How do you measure water use for this irrigation type?”
6. Click “Calculate Water Usage”
7. the calculator should look and work as before

**Checklist:**
